media/js/firefox/android.js

```diff
@@ -283,85 +288,21 @@
         'data-download-version': 'href'
     });
 
-    // send-to-device test variations
-    var sendToDeviceWidget;
-    var $sendToDeviceWidgetForm;
-    var $sendToDeviceModal;
-    var $sendToDeviceEmbeddedWidget;
-    var $sendToLinks = $('.send-to');
-
-    // data-* stored in #intro element to determine test configuration
-    // contains two properties: lang (page language) and variant ('a', 'b', 'c', '')
-    var testVars = $('#intro').data();
-
-    var initSendToDeviceForm = function() {
+    if (hasWidget) {
+        sendToDeviceWidgetTop = $('#send-to-device').offset().top;
         sendToDeviceWidget = new Mozilla.SendToDevice();
         sendToDeviceWidget.init();
-    };
-
-    // set up default behavior for send to device widget - opens in modal
-    // used for no variant (no test) and variant "a"
-    var defaultSendToDevice = function() {
-        // add class to display the widget
-        $('#send-to-device-default').removeClass('send-to-device-variant');
 
-        // reference modal wrapper for widget
-        $sendToDeviceModal = $('#send-to-modal-container');
+        // add hidden input to update basket id
+        $('#send-to-device-form').append('<input type="hidden" name="send-to-device-basket-id" value="android-embed">');
```

Comment:
As per [ADavis' comment in the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1221328#c7), we want to keep the unique basket ID for this page.

However, your question regarding translation is still **very** valid. [Asked in the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1221328#c8). Marking as do not merge until we have a resolution.